docs/documentation/release_notes/topics/25_0_0.adoc

```diff
@@ -163,6 +163,10 @@ This eliminates the need to execute the build phase and rebuild your image due t
 
 For more details, see the link:{upgradingguide_link}[{upgradingguide_name}].
 
+= High availability guide enhanced
+
+The high availability guide now contains a {section} how to configure an AWS Lambda to prevent an intended automatic failback from the Backup site to the Primary site.
```

Comment:
```suggestion
The high availability guide now contains a {section} on how to configure an AWS Lambda to prevent an intended automatic failback from the Backup site to the Primary site.
```